projects/example-app/src/app/auth/services/auth-guard.service.spec.ts

```diff
@@ -1,27 +1,31 @@
-import { TestBed, inject } from '@angular/core/testing';
-import { StoreModule, Store, combineReducers } from '@ngrx/store';
+import { TestBed } from '@angular/core/testing';
+import { Store } from '@ngrx/store';
 import { cold } from 'jasmine-marbles';
 import { AuthGuard } from '@example-app/auth/services/auth-guard.service';
-import { AuthApiActions } from '@example-app/auth/actions';
 import * as fromRoot from '@example-app/reducers';
 import * as fromAuth from '@example-app/auth/reducers';
+import * as fromLoginPage from '@example-app/auth/reducers/login-page.reducer';
+import { provideMockStore, MockStore } from '@ngrx/store/testing';
 
 describe('Auth Guard', () => {
   let guard: AuthGuard;
-  let store: Store<any>;
+  let store: MockStore<fromAuth.State>;
+  const initialState: fromAuth.State = {
+    ...({} as fromRoot.State),
```

Comment:
Do we need this line?
Asking because it seems like it doesn't to a lot 😅

Reply:
It mocks out the root state.  And since the auth state extends the root state, the root state must be defined.  Open to alternate strategies.

Reply:
I would rather see

```ts
const initialState = {
  ...stuff
} as fromAuth.State;
```

The test isn't interested in anything outside the auth state.

Reply:
sounds good, update in 903dd6c.